subworkflows/nf-core/bcl_demultiplex/main.nf

```diff
@@ -10,7 +10,7 @@ include { UNTAR      } from "../../../modules/nf-core/untar/main"
 
 workflow BCL_DEMULTIPLEX {
     take:
-        ch_flowcell     // [[id:"", lane:""],samplesheet.csv, path/to/bcl/files]
+        ch_flowcell      // channel: [ [id:"", lane:""], path(samplesheet), path(files) ]     (path to samplesheet.csv and path to bcl files)
```

Comment:
There is an extra space.
Also do we need the string inside the parentheses ? As we never specified it before.